softvis3d-frontend/src/services/sonarqube/measures/SonarQubeMeasureResponse.ts

```diff
@@ -0,0 +1,51 @@
+export interface SonarQubeMeasurePagingResponse {
+    baseComponent: SonarQubeApiComponent;
+    components: SonarQubeApiComponent[];
+    paging: SonarQubePaging;
+}
+
+export interface SonarQubeMeasureResponse {
+    baseComponent: SonarQubeApiComponent;
+    components: SonarQubeApiComponent[];
+}
+
+export interface SonarQubeApiComponent {
+    id: string;
+    key: string;
+    language?: string;
+    measures: SonarQubeMeasure[];
+    name: string;
+    path: string;
+    qualifier: SonarQubeQualifier;
+}
+
+export interface SonarQubeMeasure {
+    metric: string;
+    value?: number;
+    periods: SonarQubePeriod[];
+}
+
+export interface SonarQubePeriod {
+    index: number;
+    value: string;
+}
+
+export interface SonarQubePaging {
+    pageIndex: number;
+    pageSize: number;
+    total: number;
+}
+
+/**
+ * BRC - Sub-projects
+ * DIR - Directories
+ * FIL - Files
+ * TRK - Projects
+ * UTS - Unit Test Files
+ */
+export type SonarQubeQualifier =
```

Comment:
Option 1: Make them part of a class
```TS

export class SonarQubeApi {
    public static TYPE_SUBPROJECT = "BRC";
    public static TYPE_DIRECTORY = "DIR";
    // ....
}
```
------------
Option 2: Standalone Constants to be imported
```TS

const TYPE_SUBPROJECT = "BRC";
const TYPE_DIRECTORY = "DIR";
// ....

export {
    TYPE_DIRECTORY,
    TYPE_SUBPROJECT
    //...
}
```